management/src/test/scripts/org/kaazing/gateway/management/jmx/wse.session.with.user.principal.joe.rpt

```diff
@@ -51,10 +51,3 @@ read header "Content-Type" /(?!text\/plain;charset=windows-1252)/
 read header "Connection" "close"
 read [0x01 0x30 0x30 0xFF] # NO-OP
 
-read notify JOE_WSE_SESSION_ESTABLISHED
-
-# read close frames
```

Comment:
This is covered by our wse spec tests.  Those verify that the close works properly when the session is closed on the accept (server) side. We do not need to reverify that in this test. I took out the close frames so that the script  could be reused in sessionAttributePrincipalsShouldListUserPrincipals without having to make that test duplicate the logic of closing the sessions.